target/msm8916/init.c

```diff
@@ -115,6 +115,7 @@ void target_sdc_init()
 	struct mmc_config_data config;
 	struct mmc_config_data sd_config;
 
+#ifndef DISABLE_EMMC
```

Comment:
This looks more like "disable first emmc"? but I assumed it disables all mmc's from the name.
`USE_SECONDARY_MMC` maybe?